### ISSUE TITLE
Models + analysis

### ISSUE THREAD

**Issue by RobBlumberg** [UBC-MDS/DSCI_522_Group_302#20]
Starting this issue for all things related to creating our prediction model

**Comment by RobBlumberg** [comment]
I transformed the original training set to order=5, and then ran cross validation on the number of features to select. Then I trained a linear model with the optimized hyperparameters and got results on a test set (this test set is a subset of training_data, not the actual test_data). Sending a PR for this soon

Edit: PR sent
